packages/form-core/src/validate/ValidateMixin.js

```diff
@@ -556,7 +556,9 @@ export const ValidateMixin = dedupeMixin(
           this.__feedbackQueue.add(async () => {
             /** @type {Validator[]} */
             this.__prioritizedResult = this._prioritizeAndFilterFeedback({
-              validationResult: this.__validationResult,
+              validationResult: this.__validationResult.filter(({ type }) =>
```

Comment:
Ah, I thought you meant changing `this.__validationResult` when we spoke before. Altering the param is indeed better (then we keep the 'validation truth regardless of interaction states'), but we would have to see if this is the parameter that we expect in  `_prioritizeAndFilterFeedback`.

I will plan a story on our backlog to write Subclasser (or maybe actually Application Developer, then we need to 'promote' this api to be public) documentation for this and better explore all edge cases.
After that, we will decide whether we solve it like this or differently.

Rest of the merge request (tests) is really great, thanks 👍

Reply:
ok, perfect 👍

Reply:
I think the best solution would be to do: 
```js
_prioritizeAndFilterFeedback({ validationResult }) {
    return super._prioritizeAndFilterFeedback({
      validationResult: validationResult.filter(v => this._showFeedbackConditionFor(v.type)),
    });
  }
}
```

Almost the same, but it still gives you the freedom to override `_prioritizeAndFilterFeedback` and have access to the full `validationResult` :)